lib/simctl.js

```diff
@@ -354,7 +354,7 @@ function shutdown(params, callback) {
 			return callback(new Error(__('Unable to find Simulator %s', params.udid)));
 		}
 
-		if (sim.availability !== '(available)') {
+		if (!sim.isAvailable && sim.availability !== '(available)') {
```

Comment:
This seems to cause issues in incremental builds using Xcode 10. I think it should use the logical-or instead of the logical-and. @cb1kenobi What do you think?

Reply:
Oh, because an undefined `sim.isAvailable` is falsey. Crap. Just need a `sim.isAvailable === false`.

Reply:
That does not seem to solve it for me. Can you try to reproduce? Steps:
1. Run an app with Xcode 10 (`ti build -p ios`)
2. Repeat 1

Expected: Currently active sim is reused
Actual: `[ERROR] Invalid iOS simulator "EAA90239-AB96-481F-B08E-31C1C83BE1B5"`

Reply:
This should fix it: https://github.com/appcelerator/ioslib/pull/89